### ISSUE TITLE
RFE: Turn specific errors into warnings by config option

### ISSUE THREAD

**Issue by hroncok** [rpm-software-management/rpmlint#1335]
I'd like to make spell checking a warning, not an error, because it generates a lot of noise in Fedora, and errors make rpmlint fail.

So I was wondering whether I can achieve this in our config, and it seems to be impossible at this moment.

Would it make sense to support setting which problems are reported as errors and which are reported as warnings by the configuration?

**Comment by danigm** [comment]
As far as I can tell, right now it's possible to do [this in config](https://github.com/rpm-software-management/rpmlint/blob/main/rpmlint/filter.py#L106-L111):

```
            # If we have any badness configured then we 'stricten' and call the
            # result Error. Otherwise we downgrade the error to Warn.
            if badness > 0:
                level = 'E'
            elif level == 'E':
                level = 'W'
```

If you set the `Scoring` to `0` or lower it'll change the level to warning, something to add to the [scoring.toml](https://github.com/rpm-software-management/rpmlint/blob/main/configs/Fedora/scoring.toml) file.
